extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SchedulerRuntimeConfig.java

```diff
@@ -13,4 +15,9 @@
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
 
-}
\ No newline at end of file
+    /**
+     * Scheduled task will be flagged as overdue if next execution time is exceeded by this period.
+     */
+    @ConfigItem(defaultValue = "1")
```

Comment:
1 second is IMO a bit too strict default. Actually, now when I think about it... it might make more sense to configure the overdue per job, i.e. add something like `@Scheduled#overduePeriod()`. @machi1990 WDYT?

Reply:
Hi  @mkouba, updated the PR according to your review. should I implement this one also ?

Reply:
I'd like to hear from @machi1990 about his opinion ;-)

Reply:
I can see the usefulness of both options. Having a global one like the way we do here will enable for an overall control of the overdue for all the schedulers: it is like a default overdue duration. @mkouba you were thinking of having one or the other?

Reply:
We can have both and `@Scheduled#overduePeriod()` defaulted to the global one...

Reply:
> We can have both and `@Scheduled#overduePeriod()` defaulted to the global one...

+1, @rmanibus what do you think about this?

Reply:
Yes, it will obviously be more flexible, I will try to update the PR today and add some tests for this. For Quartz, as we do not have a global object representing the job, should I rather add the grace period in the jobdata as a primitive, or create a map storing the durations (like for the invokers) ? What will happen if I store it as a map and two instances of a clustered scheduler run a different version of the app ?

Reply:
> For Quartz, as we do not have a global object representing the job...

Hm, maybe it would make more sense to create a global map, i.e. identity -> job metadata.

Reply:
> > For Quartz, as we do not have a global object representing the job...
> 
> Hm, maybe it would make more sense to create a global map, i.e. identity -> job metadata.

+1